src/Bus/temp-bus-list.js

```diff
@@ -0,0 +1,203 @@
+const tempStopList = [
+  {
```

Comment:
Part of me thinks this kind of test data could be permanent, if we want to use it in testing (testing! amazing).